antarest/launcher/service.py

```diff
@@ -389,6 +430,132 @@ def get_log(
 
         raise JobNotFound()
 
+    def _export_study(
+        self,
+        job_id: str,
+        study_id: str,
+        target_path: Path,
+        launcher_params: Optional[JSON],
+    ) -> None:
+        with db():
+            self.append_log(
+                job_id, f"Extracting study {study_id}", JobLogType.BEFORE
+            )
+            self.study_service.export_study_flat(
+                study_id,
+                RequestParameters(DEFAULT_ADMIN_USER),
+                target_path,
+                outputs=False,
+            )
+            self.append_log(job_id, "Study extracted", JobLogType.BEFORE)
```

Comment:
JobLogType.AFTER

Reply:
nah, AFTER is for when the study is really launched, not only exported ^^'